types/ApiRequests.ts

```diff
@@ -49,10 +54,22 @@ export interface UserPatches {
   passwordChange?: PasswordUpdate;
 }
 
+export interface FeedbackPatches extends Partial<Feedback> {
```

Comment:
maybe `AcknowledgeFeedback` would be a more apt name? no need to extend `Partial<Feedback>` either

Reply:
I extended `Partial<Feedback>` here to allow editing title/description as well as acknowledged, but yeah doesn't look like editing feedback after submission is on the figma